src/main/java/walle/WallE.java

```diff
@@ -0,0 +1,220 @@
+package walle;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WallE {
+    private static final int MAX_LIST_SIZE = 100;
+    private static final String exitMessage = "Bye. Hope to see you again soon!";
+    private static final String greeting = "Hello! I'm Wall-E!\n" + "\tWhat can I do for you?\n\n";
+    private static int listSize = 0;
+    private static final String FILE_NAME = "data.txt";
```

Comment:
`exitMessage` and `greeting` could be considered constants since they technically shouldn't change in the course of the program
> Constant names must be all uppercase using underscore to separate words (aka SCREAMING_SNAKE_CASE)